Xamarin.Forms.ControlGallery.iOS/Xamarin.Forms.ControlGallery.iOS.csproj

```diff
@@ -22,7 +22,7 @@
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
     <MtouchArch>i386, x86_64</MtouchArch>
-    <MtouchLink>SdkOnly</MtouchLink>
+    <MtouchLink>None</MtouchLink>
```

Comment:
We should revert this change

Reply:
VS shows a warning that Hot reload will not work. Why do we use SdkOnly for debug?

Reply:
Better to make sure sdk linker works when we run tests. Please revert